__tests__/add-remove.test.js

```diff
@@ -0,0 +1,99 @@
+import { addNewTodo, onDeleteTodo } from '../src/add-remove.js';
+import { localStorageMock } from '../__mocks__/mockStorage.js';
+
+describe('Todo', () => {
+  describe('Add Todo', () => {
+    test('task 1 should be equal {description; task 1, completed: false, index: 1} ', () => {
+      const task = 'task 1';
+      const todos = [];
+      const expectedTodo = { description: 'task 1', completed: false, index: 1 };
+      const newTodo = addNewTodo(task, todos);
+      expect(newTodo).toEqual(expectedTodo);
+    });
+  });
+
+  describe('Delete Todo', () => {
+    test('task 1 should be equal {description; task 1, completed: false, index: 1} ', () => {
+      let todos = [];
+      const expectedTodo = { description: 'task 1', completed: false, index: 1 };
+
+      todos = onDeleteTodo(expectedTodo.id, todos);
+
+      expect(todos.length).toEqual(0);
+    });
```

Comment:
- [ ] I would recommend to add more test cases for `add` and `delete` todo's so that it will cover all the edge test case as well, like ` where there is more than 2 todo's and deleting 1 will delete only 1 and rest will remain there.`